.github/workflows/create-release.yaml

```
@@ -99,6 +103,14 @@ jobs:
           RELEASE_ID=$(./.github/scripts/create-draft-release.sh ${{ github.event.inputs.name }})
           echo "release_id=$RELEASE_ID" >> $GITHUB_OUTPUT
 
+      - name: Render module-config
+        env:
+          MODULE_VERSION: ${{ github.event.inputs.name }}
+        run: |
+          make community-module-config
+          echo "==== module-config ==="
```

```suggestion
          echo "==== community-module-config ==="
```

Lets not mix naming for two cases: community vs submission to control-plane